lib/meson.build

```diff
@@ -0,0 +1,39 @@
+###################################################################################
```

Comment:
As for license - see my previous comment.

As for the file itself - do they have to be in every subfolder, or could we settle for a single `mason.build` file in the project's root (or the library's root)?

Reply:
Sorry about that, I wrote a Meson file generation tool that would scan a project and add meson.build files depending on structure, there was only supposed to be short comments with a path to the file starting from root in the sub files.

Reply:
I will also fix that.

Reply:
I think I fixed all of the the licensing information.

Reply:
I would still prefer if the license text was removed. Simply use:

```
License: MIT
```

No need to include the license text.

If you don't have time for this, no worries, I can clean it up after the merge.

Reply:
Got it making one more edit.

Reply:
Ok I switched the long verbose version of the license info to use this suggested one instead.